src/main/app/src/stores/haku-store.js

```diff
@@ -309,6 +309,19 @@ class HakuStore {
     }
   }
 
+  @action
+  clearFilters = () => {
+    this.filter.koulutus = [];
+    this.filter.paikkakunta = '';
+    this.filter.kieli = [];
+    this.filter.koulutustyyppi = [];
+    this.filter.koulutusala = [];
+    this.filter.opetusKielet = [];
+    this.filter.paikkakunta = [];
+    this.filter.sijainti = [];
+    this.filter.selectedsijainnit = [];
```

Comment:
Itse ehkä olisin laittanut nuo filtterien default arvot johonkin vakio-objektiin ja sitten tässä vaan mergennyt sen this.filterin kanssa, niin ei tarvis toistaa this.filter. Samaa vakiota vois myös käyttää filtterin alustamiseen alunperin.